docgen/assets/js/sidebar.js

```diff
@@ -105,7 +105,7 @@ function toggleDocumentationSidebar() {
   function init() {
     const bodySize = document.body.clientWidth;
     if (bodySize <= 960 && sidebarNav) {
-      trigger.addEventListener('click', () => {
+      trigger.addEventListener('click', (e) => {
```

Comment:
If you don't use the event, you should remove it from the arguments of the function.